pkg/kubectl/metricsutil/metrics_printer.go

```
@@ -87,7 +91,33 @@ func (printer *TopCmdPrinter) PrintNodeMetrics(metrics []metricsapi.NodeMetrics,
 	return nil
 }
 
-func (printer *TopCmdPrinter) PrintPodMetrics(metrics []metricsapi.PodMetrics, printContainers bool, withNamespace bool, noHeaders bool) error {
+func sortNodesByMetrics(metrics []metricsapi.NodeMetrics, sortBy string) {
+	var usages = make([]v1.ResourceList, len(metrics))
+	for i, v := range metrics {
+		if err := scheme.Scheme.Convert(&v.Usage, &usages[i], nil); err != nil {
+			usages[i] = nil
```

We can't silently pass errors, if there's a problem we should error out.